test/unit/spec/EnrollUser_spec.js

```diff
@@ -0,0 +1,158 @@
+/* eslint max-params: [2, 13], max-len: [2, 160] */
+define([
+  'okta',
+  '@okta/okta-auth-js/jquery',
+  'util/Util',
+  'helpers/mocks/Util',
+  'helpers/dom/EnrollUserForm',
+  'helpers/util/Expect',
+  'LoginRouter',
+  'sandbox',
+  'helpers/xhr/PROFILE_REQUIRED_UPDATE',
+  'helpers/xhr/PROFILE_REQUIRED_NEW',
+  'helpers/xhr/SUCCESS',
+  'helpers/xhr/UNAUTHENTICATED_IDX'
+],
+function (Okta, OktaAuth, LoginUtil, Util, EnrollUserForm, Expect, Router,
+  $sandbox, resProfileRequiredUpdate, resProfileRequiredNew, resSuccess, resUnauthenticatedIdx) {
+
+  var { _, $, Backbone } = Okta;
+  var itp = Expect.itp;
+
+  function setup (settings, resUnauthenticatedIdx) {
+    settings || (settings = {});
+    var successSpy = jasmine.createSpy('successSpy');
+    var setNextResponse = Util.mockAjax();
+    var baseUrl = 'https://example.okta.com';
+    var logoUrl = 'https://logo.com';
+    var authClient = new OktaAuth({ url: baseUrl, transformErrorXHR: LoginUtil.transformErrorXHR });
+    var router = new Router(_.extend({
+      el: $sandbox,
+      baseUrl: baseUrl,
+      features: { consent: true },
+      logo: logoUrl,
+      authClient: authClient,
+      globalSuccessFn: successSpy
+    }, settings));
+    Util.registerRouter(router);
+    Util.mockRouterNavigate(router);
+    Util.mockJqueryCss();
+
+    router.refreshAuthState('dummy-token');
+    settings = {
+      router: router,
+      successSpy: successSpy,
+      form: new EnrollUserForm($sandbox),
+      ac: authClient,
+      setNextResponse: setNextResponse
+    };
+    if (resUnauthenticatedIdx) {
+      setNextResponse(resUnauthenticatedIdx);
+      return Expect.waitForPrimaryAuth(settings);
+    } else {
+      setNextResponse(resProfileRequiredUpdate);
+      return Expect.waitForEnrollUser(settings);
+    }
+  }
+
+  Expect.describe('Enroll User', function () {
+    Expect.describe('Enroll User Form', function () {
```

Comment:
`Expect.describe` has done extra setup hence running one `Expect.describe` at top level is sufficient.